projects/assets/src/translations/en/organization.ts

```diff
@@ -195,6 +195,19 @@ export const organization = {
     },
     currency: 'Currency',
   },
+  orderApprovalsList: {
```

Comment:
In general ion the PR, seems more consistent to use the singular form `approval` everywhere if possible.
Lets discuss.